src/main/java/serialization/PersonService.java

```diff
@@ -0,0 +1,90 @@
+package main.java.serialization;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class PersonService {
+
+    public static String getHeaderCSVRaw() {
+        return "Id,Name,Age,City";
+    }
+
+    public List<Person> getAllPersons() {
+        List<Person> persons = new ArrayList<>();
+
+        Person person1 = new Person(1, "Maga", 22, "Kyiv");
+        Person person2 = new Person(2, "Sasha", 18, "Kyiv");
+        Person person3 = new Person(3, "Tima", 28, "Kyiv");
+        Person person4 = new Person(4, "Yarik", 12, "Kyiv");
+
+        persons.add(person1);
+        persons.add(person2);
+        persons.add(person3);
+        persons.add(person4);
+
+        return persons;
+    }
+
+    public String getYoungestPerson(List<Person> personsList) {
```

Comment:
Do not place the type of the variable into it title. Like 'personsList', simply 'persons'.
(if variable is in plural, that usually means that it is an array or set)